pkg/auth/login_credentials_manager.go

```diff
@@ -323,6 +375,20 @@ func (h *LoginCredentialsManagerImpl) isSSOUser(email, organizationId string) bo
 	return err == nil && res.GetIsSso()
 }
 
+func (h *LoginCredentialsManagerImpl) isOnPremSSOUser(url, caCertPath string, unsafeTrace bool) bool {
+	clientManager := &MDSClientManagerImpl{}
```

Comment:
Sort of. I'll have to look into this again because I don't remember the specifics, but I did this to avoid using the mocked version for one reason or another.